ios/StatusIm/Info.plist

```diff
@@ -82,6 +82,8 @@
 	<string>Photos access is required to give you the ability to save images.</string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Photos access is required to give you the ability to send images.</string>
+	<key>NSAppleMusicUsageDescription</key>
+	<string>Play audio messages</string>
```

Comment:
should  be `A message that tells the user why the app is requesting access to the user’s media library`.

Reply:
@siddarthkay and do we really use the music lib

Reply:
I mean I am not really aware of why we use the media lib, do you know at least i could write a more informative message

Reply:
my assumption is most likely because we request permission to media library here : 
https://github.com/status-im/status-mobile/blob/555959938152ac65a0b860acf4b165f6f05c49c9/ios/Podfile#L19
media includes both images + music.

Reply:
I see, its not just music. thats right

Reply:
you could remove this permission and see if things work fine on iOS where we access media like photos etc.
If it works then we may not need to add this in our info.plist

Reply:
it was added in `react-native-permissions` library here -> https://github.com/zoontek/react-native-permissions/pull/430/commits/001f633ec8e4e0b4b7c01bce9ce99ddd94300a5b which strengthens my assumption that media permission require us to add `NSAppleMusicUsageDescription ` in `info.plist`

Reply:
I will check it out

Reply:
I am currently looking to see if we actually need the permission as you have suggested, this pr however should be ready, you could approve it so I can run the e2e tests in the meantime

Reply:
hmm e2e tests wont help us though. since this is iOS specific. e2e will only validate Android.

Reply:
better to ask @status-im/mobile-qa to take a look at PR build on iOS after you make the changes